### ISSUE TITLE
Not really an issue...

### ISSUE THREAD

**Issue by kababoom** [zearp/Nucintosh#92]
Actually just opening this to say thank you...

Especially since there are some useless trolls around I want to let you know I do appreciate all the unpaid time you spent creating and maintaining this repo 😉 

Cheers,
-M

**Comment by sureshsharmaxyz** [comment]
I have been wanting to do the same, but I was hesitant to open an issue for a non-issue. So let me add to this thread by saying, thanks a lot - your work is very much appreciated!
(And I hope the entitled trolls don't discourage you too much!)

**Comment by gtrooper** [comment]
Guys, the work of Zearp is simply PriceLess ! He is an amazing person and we owe a lot to him ! Please, ignore and block all the stupid trolls in the internet. They are just too many ! Never forget what Einstein said YOU CANT FIGHT STUPIDITY! Ignore the n00bs Zearp ! You are a true Legend !!!

**Comment by vardumper** [comment]
Absolutely! I'll join in, here. I also appreciate your work, Zearp. Btw: I'm also using my NUC for work on a daily basis. 64GB RAM, 1TB NVMe, and really happy with it. 
Btw: Here's some of my experience. When I bought my NUC8 i7 version I ordered the Akasa Turing case. Unfortunately during gaming (which I do occasionally on an external Win SSD) the case simply got way too hot. I switched back to the OEM case. Then I had to change a broken OEM fan. When the replacement broke as well, I decided to stop replacing the fan, and switched back to the Akasa case. I now have an external USB 12cm fan which I can turn to low/mid/high manually whenever there's some workload which I prefer to replacing internal fans.

**Comment by spyv007** [comment]
Thanks from me too! I use your distro many hours every day and it's been
rock solid. Bravo, and thanks again.

**Comment by zearp** [comment]
Thanks for the kind words! 😳

I'm glad people can use the EFI to get started with hackintosh and macOS. These NUCs make perfect little Mac mini replacements. Specially when the prices were still nice. At one point Amazon was selling the i5 model for about $250. I hope the prices will normalise again but for hackintosh on NUC it may not matter as anything newer than 10th gen can't be used for macOS due to incompatible iGPU. Spoofing it might work but I don't want to use such tricks to get something as important as accelerated graphics to work. The iGPU should be natively supported. 10th gen NUC have a weaker iGPU (UHD 620) than 8th gen (Iris Plus 655). I don't know what Intel was thinking but only the CPU is better in the 10th gen models. The 10th models have 2 extra cpu cores and is slightly faster.

Regarding temps I think the i7 gets a bit hotter than the i5 models. You can lower temps by reducing the TDP a bit in the bios and/or with VoltageShift on the fly to experiment with it. Reducing TDP will impact performance so I would try to undervolt first as no performance will be lost, you could even gain some performance as it may be able to boost longer with reduced temps.

It will also help to disable turbo boosting completely for some games, depending on the game this might make no difference performance while still lowering temps. When I was testing the iGPU a while ago I had to disable turbo boosting for cpu intensive games like City Skylines. With turbo boosting disabled the game still performed fine and wouldn't crash anymore. YMMV.

Paste can also make a difference. I have a stock i5 NUC here that I didn't re-paste and it gets over 90c so fast! Reminds me of my MacBook. With new paste applied it takes lot longer for it to go over 90c. My Akasa case can get really hot too, but that's a good sign. Heat needs to be transferred to the whole case. Mine got about 50-60c under sustained load when I checked it with an infra red temperature meter. The cpu usually tops out around 70-75c with synthetic loads. That's with no changes in TDP and a little undervolting.

I was thinking of a little mod to help the convection by adding a thin silent 40mm fan to the bottom of the case when it stands up straight. It would be easy to do with some long screws and spacers so it can get some fresh air. It wouldn't need to spin at a high rpm, just get the air moving more will help a lot. You could power it with one of the internal usb 2 headers and tidy the wires up. A different location on the desk with more draft could also help. 

I'm looking to find a nice price for the other NUC8 compatible case Akasa makes, the Plato X8. It has more surface area by the looks of it so might be more suitable for i7 and sustained loads.

[A-NUC43-M1B_g01](https://user-images.githubusercontent.com/63272687/185376587-89fcc8ab-f345-46ec-a5ed-aa553328ce20.jpg)

Replacing the stock cpu fan might be more tricky than it looks. But I don't know if Intel uses any special sauce like Dell does when it comes to PWM and controlling the fans. In my Dell hacks replacing fans can lead to having to press F2 on every boot cuz the system detects it's not a proper replacement fan. I had to make a custom cable to trick the bios into not giving me the silly warning each boot.

Improving the air intake of the stock case can help too. I heard people had good results removing the 2.5" drive bay if not in use. There might be more ways to improve the airflow in the stock case. The main issue is that the fan is at the bottom and the intake of the fan is blocked by the bottom of the case. Cutting a round hole at the bottom would be best but also destructive. Any other things like removing drive bay just allow for some more airflow but the fn is on the other side of the board so it may not get that much extra cool air. I've tried to find some diagram or tech info the airflow/cooling solution designed by Intel but can't find anything useful.

![nuc8i3beh_fan](https://user-images.githubusercontent.com/63272687/185378724-dfaf55f6-a85b-46c8-b71a-eeb28cdad558.jpg)

By the looks of it the only place the fan takes in air is the round opening. Which faces the bottom of the machine when in use. It will take in air from mesh in the sides only. A round hole in the bottom plate covered with some mesh or standard fan grill will allow it to get a lot more air directly into the fan. Too bad it's tricky to find bottom plate replacements. I would like to order one to play with and keep the stock one in tact.

tl;dr I will keep updating the EFI until my NUCs all break down or simply become too slow/outdated for the use cases.

ps: Reddit has a nice NUC community where you can find a lot of information and inspiration. Many cooling/fan mods too! Like this one: https://www.reddit.com/r/intelnuc/comments/wod5p2/intel_nuc_fan_mod/

**Comment by kababoom** [comment]
Good to see you still got inspiration looking for NUC hardware improvements and future usage..

Thanks to all who joined and showed their appreciation, I think the message was received so we can closed this thread 🙂
